src/public/style.css

```diff
@@ -1,5 +1,5 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600;700;800&display=swap');
-@import './preflight.css';
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500;600;700;800&display=swap");
+@import "./preflight.css";
```

Comment:
Prettier poprawia mi to od razu na pojedyncze (i z tego co widzę powinien, bo w `.prettierrc` mamy `  "singleQuote": true`), zobacz czy masz włączonego Prettiera dla CSS.